extension/src/popup/views/Account.tsx

```diff
@@ -164,5 +166,5 @@ export const Account = () => {
       </AccountEl>
       <Footer />
     </>
-  );
+  ) : null;
```

Comment:
Now we'll just be seeing the Loading screen until this component returns something